azure-iot-device/tests/iothub/pipeline/conftest.py

```diff
@@ -8,4 +8,6 @@
     fake_pipeline_thread,
     fake_non_pipeline_thread,
     unhandled_error_handler,
+    arbitrary_op,
+    arbitrary_event,
```

Comment:
do these actually need to be here?  (and in tests/provisioning/pipeline)  I assume this is bleed through from another change that didn't make it into this PR, but I wouldn't rule out some weird fixture thing causing this to suddenly be necessary.

Reply:
I think this is bleedthrough yeah - right now I believe these are only needed in common tests, but they will be needed here shortly as well (I reverted some in-progress changes in this package in order to get this PR completed).

---
In reply to: [338219175](https://github.com/Azure/azure-iot-sdk-python/pull/369#discussion_r338219175) [](ancestors = 338219175)